src/customsdpCpp.cpp

```diff
@@ -446,6 +447,11 @@ int custom_sdpCpp(
            diagO,bestx,besty,bestz,Zi,O,rhs,dZ,dX,dy,dy1,Fp,
            printlevel,params);
 
+		if (i % 500 == 0)
```

Comment:
so this is definitely fine, I just wonder if the 500 shouldn't be a function of the number of samples. Maybe you want to run the callback after the maximum of 100 iterations or 1% of the samples? So something like `std::min(int(0.01 * total_no_samples), 100);`?

Reply:
I guess I could just make it so that I have only five ticks for the progress bar, irrespective of no samples

Reply:
that also depends on the number if chains though, and that is not really passed to the glb.

Reply:
I altered it, so that there are always 4 ticks per chain no matter how many samples.